assessments/unit-01/code.js

```diff
@@ -5,3 +5,107 @@ const emit = (a, b, product) => {
 };
 
 // Write your code here ...
+const averageWeight = (totalWeight, numItems) => {
+  return (totalWeight/numItems)
+}
+
+const hypotenuse = (a, b) => {
+  return Math.sqrt(a**2 + b**2)
+}
+
+const maxRadius = (widthOfArea, heightOfArea) => {
+  return (Math.min (widthOfArea, heightOfArea))/2
+}
+
+const numCircles = (radius, widthOfArea) => {
+  return Math.floor (widthOfArea/radius)
```

Comment:
As a matter of style, should be no space between the function name and the paren. And spaces around (most) operators

```suggestion
  return Math.floor(widthOfArea / radius)
```